_i18n/en/documentation/getting-started/home.md

```diff
@@ -0,0 +1,18 @@
+The Home screen allows you to have a quick overview of your podcasts, start listening and perform a range of actions, all conveniently from the same screen. It contains five sections, each of which displays episodes or subscriptions to interact with directly, but also provides a shortcut to the related screen (e.g. your queue):
+
+## Continue listening
+This section allows you to quickly and easily carry on listening to episodes you already have in your queue, beginning with those you have already started listening to, but haven’t finished just yet.
+
+## See what’s new
+In this section, you will find the latest episodes that have landed in your inbox as a result of having refreshed your subscriptions.
+
+## Get surprised
+Here you can find a random selection of episodes from your subscriptions you haven’t listened to yet. If you don’t see anything you feel like listening, you can tap the arrow button to refresh the selection.
+
+## Check your classics
+Here you can quickly see and get access to your all-time most-listened subscriptions.
+
+## Manage downloads
+In this section, you can interact with the episodes you downloaded on your device.
+
+You can customise your Home screen – that is, reorder these sections or hide some of them – by tapping `…` in the top bar and selecting `Configure Home Screen`.
```

Comment:
How about moving this sentence up to the top? Then it is more clear that it does not belong to the downloads section.

Could you please switch to the American English spelling of "customize" to me more consistent with the remaining website?